templates/CRM/MembershipExtras/Page/CurrentPeriodTab.tpl

```diff
@@ -1,5 +1,6 @@
 <script>
   var currentFinancialTypes = JSON.parse('{$financialTypes|@json_encode}');

```

Comment:
This statement `JSON.parse('{$financialTypes|@json_encode}');` is redundant
`var currentFinancialTypes = {$financialTypes|@json_encode}` alone is sufficient